src/anim/evaluator/anim-target-value.js

```diff
@@ -58,17 +85,37 @@ class AnimTargetValue {
     updateValue(index, value) {
         // always reset the value of the target when the counter is 0
         if (this.counter === 0) {
-            this.value[0] = 0;
-            this.value[1] = 0;
-            this.value[2] = 0;
-            this.value[3] = 1;
+            AnimEvaluator._set(this.value, [0, 0, 0, 1], this.valueType);
```

Comment:
don't construct the constant array here, have it once outside of the class

Reply:
Done 👍